modules/ephemeral-storage-using-a-sharedconfimap-object-in-a-pod.adoc

```diff
@@ -0,0 +1,82 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod_{context}"]
+= Using a SharedConfigMap instance in a pod
+
+[role="_abstract"]
+To access a `SharedConfigMap` custom resource (CR) instance from a pod, you grant a given service account RBAC permissions to use that `SharedConfigMap` CR instance.
+
+.Prerequisites
+
+* You have created a `SharedConfigMap` CR instance for the config map that you want to share across namespaces in the cluster.
+* You must have permission to perform the following actions:
+** Discover which `SharedConfigMap` CR instances are available by entering the `oc get sharedconfigmaps` command and getting a non-empty list back.
+** Determine if the service account your pod specifies is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.
+** Determine if the service account your pod specifies is allowed to use `csi` volumes, or if you, as the requesting user who created the pod directly, are allowed to use `csi` volumes.  See "Understanding and managing pod security admission" for details.
+
+[NOTE]
+====
+If neither of the last two prerequisites in this list are met, create, or ask someone to create, the necessary role-based access control (RBAC) so that you can discover `SharedConfigMap` CR instances and enable service accounts to use `SharedConfigMap` CR instances.
+====
+
+.Procedure
+
+. Grant a given service account RBAC permissions to use the `SharedConfigMap` CR instance in its pod by using `oc apply` with YAML content.
++
+[NOTE]
+====
+Currently, `kubectl` and `oc` have hard-coded special case logic restricting the `use` verb to roles centered around pod security. Therefore, you cannot use `oc create role ...` to create the role needed for consuming a `SharedConfigMap` CR instance.
```

Comment:
It looks like the `builds-docs-main` branch's `common-attributes.adoc` file doesn't have this set, but for references to `oc`, it should be "OpenShift CLI (`oc`)", as seen in the following line from the [`main` branch's `common-attributes.adoc`](https://github.com/openshift/openshift-docs/blame/main/_attributes/common-attributes.adoc):

```
:oc-first: pass:quotes[OpenShift CLI (`oc`)]
```

For now, suggest just manually typing it out, but also would be good to get that into the builds' branches' available attributes eventually.

Also:
* Add comma
* Don't think it needs the `...` in the command reference
```suggestion
Currently, the `kubectl` command and the OpenShift CLI (`oc`) have hard-coded, special case logic restricting the `use` verb to roles centered around pod security. Therefore, you cannot use `oc create role` commands to create the role needed for consuming a `SharedConfigMap` CR instance.
```